### PR TITLE
chore: turn off prop types warning for ts files FE-6085

### PR DESCRIPTION
**Pull request by edleeks87**
### Proposed behaviour

Amends eslint config to remove warning for prop-types in TS files

Migrates `Alert` component tests from cypress to playwright:
- deletes `alert.cy` file,
- deletes any unused locators

### Current behaviour

component testing is done by cypress for `Alert`
lint warning for missing prop types in TS files

### Checklist

- [x] Commits follow our style guide
- [x] Cypress automation tests added or updated if required
- [x] Playwright automation tests added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

File: src/components/alert/component.test-pw.tsx
Comment (DlgSHi) [inline src/components/alert/component.test-pw.tsx:1]: can we rename the file to be consistent and name it as `components.test-pw.tsx`?

File: src/components/alert/alert.pw.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    await checkDialogIsInDOM(page);
+    await page.keyboard.press("Escape");
+    await checkDialogIsInDOM(page);
+  });
+
+  test("with keyboard accessible close icon button which closes the dialog when enter key is pressed", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<AlertComponent title="title">Alert</AlertComponent>);
```
Comment (DlgSHi) [inline src/components/alert/alert.pw.tsx:79]:
```suggestion
    await mount(<AlertComponent title="title">Alert</AlertComponent>);
    
```

File: src/components/alert/alert.pw.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    await expect(cross).toBeFocused();
+    await cross.press("Enter");
+    await checkDialogIsNotInDOM(page);
+  });
+
+  test("with close icon button that closes dialog when clicked", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<AlertComponent title="title">Alert</AlertComponent>);
```
Comment (DlgSHi) [inline src/components/alert/alert.pw.tsx:92]:
```suggestion
    await mount(<AlertComponent title="title">Alert</AlertComponent>);
    
```

File: src/components/alert/alert.pw.tsx (hunk 1 above)
Comment (edleeks87) [reply src/components/alert/alert.pw.tsx:79]: it's the same 🤷‍♂️

**Review by DlgSHi — APPROVED** [review]

**Review by tomdavies73 — APPROVED** [review]

**Pull request merged**